src/components/kyb/OwnershipStructureStep.tsx

```
@@ -0,0 +1,362 @@
+import { useState, useEffect } from 'react';
+import { Plus, Trash2, AlertCircle } from 'lucide-react';
+import { BusinessData } from '@/app/kyb/page';
+
+interface OwnershipStructureStepProps {
+  data: BusinessData;
+  updateData: (updates: Partial<BusinessData>) => void;
+}
+
+export default function OwnershipStructureStep({ data, updateData }: OwnershipStructureStepProps) {
+  const [uboCounter, setUboCounter] = useState(1);
```

ESLint

[error] 'uboCounter' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)